pages/common/webpack.md

```diff
@@ -10,10 +10,10 @@
 
 `webpack {{app.js}} {{bundle.js}} --module-bind 'css=css'`
 
-- Pass a config file and show compilation progress:
+- Pass a config file and show compilation progress. (As an example config, you can export an object containing `entry` and `output.filename` properties, which correspond to the entry and output file names):
```

Comment:
I was envisioning something simpler, say: "Pass a config file (with e.g. the entry script and output filename) and show compilation progress:". What do you think?